client.go

```diff
@@ -48,54 +47,54 @@ var (
 )
 
 const (
-	// DefaultAddress is the default IPV4 address that is used for a connection
-	DefaultAddress = "127.0.0.1"
-	// DefaultPort is the default port used for a connection
-	DefaultPort     = 6640
-	UNIX            = "unix"
-	TCP             = "tcp"
-	SSL             = "ssl"
-	SKIP_TLS_VERIFY = true
+	DefaultTCPAddress  = "127.0.0.1:6640"
```

Comment:
I think it is better to add protocol here, i.e. "tcp:127.0.0.1:6640"

Reply:
ok